src/rat/run_rat.py

```diff
@@ -88,6 +90,11 @@ def run_rat(config_fn, operational_latency=None):
             config_copy = copy.deepcopy(config)
             # Run RAT for basin
             no_errors, latest_altimetry_cycle = rat_basin(config, log)
+            # Run RAT forecast for basin if forecast is True
+            if config['PLUGINS']['forecast']:
+                log.info('############## Starting RAT forecast for '+config['BASIN']['basin_name']+' #################')
+                no_errors = forecast(config, log)
+                log.info('############## RAT forecast finished for '+config['BASIN']['basin_name']+ ' with '+str(no_errors)+' error(s). #################')
```

Comment:
These lines can be added to just above line 169 of the updated file (that includes forecast) or above line 162 for the file in main branch. This will ensure that forecast mode can run for multiple basins as well. Also, make sure that forecast_no_errors is used.